### ISSUE TITLE
vmagent: cannot remote-write to an ipv6-only victoriametrics host

### ISSUE THREAD

**Issue by dekonnection** [VictoriaMetrics/VictoriaMetrics#469]
**Describe the bug**

With vmagent, when trying to remote-write to a victoriametrics instance located behind an ipv6, even with `--enableTCP6`, the following error is encountered:

```
2020-05-09T19:02:38.743Z	error	VictoriaMetrics/app/vmagent/remotewrite/client.go:257	couldn't send a block with size 232 bytes to "https://victoriametrics.my.domain:8428/api/v1/write": couldn't find DNS entries for the given domain. Try using DialDualStack; re-sending the block in 60.000 seconds
```

Where `victoriametrics.my.domain` only has an AAAA record.

Same thing when passing the ip directly:

```
2020-05-09T19:07:22.277Z	error	VictoriaMetrics/app/vmagent/remotewrite/client.go:257	couldn't send a block with size 232 bytes to "http://[2a01:****:***:****:****::17]:8428/api/v1/write": couldn't find DNS entries for the given domain. Try using DialDualStack; re-sending the block in 32.000 seconds
```

Since ipv6-only scraping only works since the latest release, with `--enableTCP6`, I think support for ipv6-only remote-write has not yet been introduced.

**To Reproduce**
Configuring a vmagent instance to scrape some hosts, and remote-write to a victoriametrics instance via its ipv6 address.
(either by AAAA-only DNS record, or with the IP directly) 

**Expected behavior**
Being able to use an ipv6-only remote-write target :)

**Version**

```
$ docker exec -it vmagent_vmagent_1 /vmagent-prod --version
vmagent-20200507-201711-tags-v1.35.3-0-gefc8e3c5
```

**Used command-line flags**
```
--enableTCP6 --promscrape.config=/storage/prometheus.yaml --remoteWrite.url=http://victoriametrics.my.domain:8428/api/v1/write"
```

**Comment by valyala** [comment]
Confirmed the bug. The bugfix has been pushed in the commit 6273385618b3a9ef3f37e1e7022c4c4cbb38358e . This commit will be included in the next release of VictoriaMetrics.

@dekonnection , could you build `vmagent` from the latest commit on the `master` branch according to [these docs](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/app/vmagent/README.md#how-to-build-from-sources) and verify whether the issue is fixed?

**Comment by dekonnection** [comment]
Oh, great !
I will try that now and keep you informed :)

**Comment by dekonnection** [comment]
Just tested with the same config and the latest commit, and it works !

Many thanks for your quick answer and fix for this issue :)

Have a good evening

**Comment by valyala** [comment]
FYI, the bugfix for ipv6 addresses in `-remoteWrite.url` has been included in [v1.35.4](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
